touch.go

```diff
@@ -10,8 +10,12 @@ import (
 )
 
 func getTouchFilename(image string) string {
+	return getHomeFileName(util.EncodeBase64Sha1(image))
```

Comment:
There's still a whole bunch of duplicated logic and no clear distinction where the code should go (this file or update_*.go). I suggest putting all homedir handling to a file called `homedir.go`. You could then have simple commands that only need the filename as parameter:
Read
Write (This can replace touch)